lib/utils.js

```diff
@@ -9,8 +9,10 @@ exports.defineCachedRelations = defineCachedRelations;
 exports.sortObjectsByIds = sortObjectsByIds;
 exports.setScopeValuesFromWhere = setScopeValuesFromWhere;
 exports.mergeQuery = mergeQuery;
+exports.createPromiseCallback = createPromiseCallback;
 
 var traverse = require('traverse');
+var Promise = require('bluebird');
```

Comment:
Should I take bluebird out of the npm dependencies?  Or add it to optionalDependencies, maybe?